From 4d5aed9acaf49a74dca6e828895c62392691e13b Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 11 Jan 2024 10:22:42 +0100
Subject: [PATCH 0426/1695] dts: stm32: enable IWDG1 on stm32mp25 boards

Configure and enable IWDG1 watchdog in OP-TEE for stm32mp25 boards.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 7a34588669695ca01cce9ab70ad7335f0f75ec33 dts: stm32: enable IWDG1 on stm32mp25 boards]
Change-Id: Ie3de7154135236b914b57ac7267dfeeb83ec9065
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts  | 5 +++++
 core/arch/arm/dts/stm32mp257f-ev1.dts | 7 +++++--
 2 files changed, 10 insertions(+), 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index 903714425..39d0dfa93 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -39,6 +39,11 @@
 	};
 };
 
+&iwdg1 {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &risaf2 {
 	status = "okay";
 };
diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 013621469..d1b4f2aa8 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -47,6 +47,11 @@
 	};
 };
 
+&iwdg1 {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &ommanager {
 	memory-region = <&mm_ospi1>;
 	memory-region-names = "mm_ospi1";
@@ -60,8 +65,6 @@
 	status = "okay";
 };
 
-
-
 &risaf2 {
 	status = "okay";
 };
-- 
2.47.2

